src/invidious/routes/feeds.cr

```diff
@@ -143,32 +143,21 @@ module Invidious::Routes::Feeds
   # RSS feeds
 
   def self.rss_channel(env)
-    locale = env.get("preferences").as(Preferences).locale
-
     env.response.headers["Content-Type"] = "application/atom+xml"
     env.response.content_type = "application/atom+xml"
 
     ucid = env.params.url["ucid"]
```

Comment:
Please add a sanity check on this user input, like `.matches?(/^\w+$/)`. You're using it directly in the generated response, which could cause an XSS vulnerability.

Reply:
As channel IDs can also contain `-` (just like video IDs and the protobuf payloads, they use use base64url), this would be a more appropriate regex `.matches?(/^[\w-]+$/)` :).